fenix/app/src/main/java/org/mozilla/fenix/translations/automatic/AutomaticTranslationOptions.kt

```diff
@@ -0,0 +1,113 @@
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations.automatic
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.compose.button.RadioButton
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Firefox Automatic Translation Options fragment compose view.
+ *
+ * @param selectedOption Selected option that will come from Gecko.
+ */
+@Composable
+fun AutomaticTranslationOptions(
+    selectedOption: AutomaticTranslationOption,
+) {
+    val optionsList = arrayListOf(
+        AutomaticTranslationOption.OfferToTranslate(),
+        AutomaticTranslationOption.AlwaysTranslate(),
+        AutomaticTranslationOption.NeverTranslate(),
+    )
+    val selected = remember { mutableStateOf(selectedOption) }
+    Column(
+        modifier = Modifier
+            .background(
+                color = FirefoxTheme.colors.layer1,
+            ),
+    ) {
+        LazyColumn {
+            items(optionsList) { item: AutomaticTranslationOption ->
+                AutomaticTranslationOptionRadioButton(
+                    text = stringResource(item.titleId),
+                    subText = stringResource(
+                        item.summaryId.first(),
+                        stringResource(item.summaryId.last()),
+                    ),
+                    selected = selected.value == item,
+                    onClick = {
+                        selected.value = item
+                    },
+                )
+            }
+        }
+    }
+}
+
+/**
+ * Top level radio button for the automatic translation options.
+ *
+ * @param text The main text to be displayed.
+ * @param subText The subtext to be displayed.
+ * @param selected [Boolean] that indicates whether the radio button is currently selected.
+ * @param onClick Invoked when the radio button is clicked.
+ */
+@Composable
+fun AutomaticTranslationOptionRadioButton(
+    text: String,
+    subText: String,
+    selected: Boolean = false,
+    onClick: () -> Unit,
+) {
+    Row(
+        modifier = Modifier
+            .padding(start = 16.dp, top = 6.dp, bottom = 6.dp, end = 16.dp)
+            .clickable { onClick() },
+    ) {
+        RadioButton(
```

Comment:
We should take a look at the accessibility on this component a little closer. 

I was correctly able to make a selection by activating the text in the list  TalkBack. (The other areas worked great too!) However, I was also able to navigate focus to the RadioButton itself as an unlabled component. 

We should follow the same pattern as we did on the switch, where the list item is selectable, but the radio button alone is not focusable or else knows it is connected.

Reply:
Current behavior:
* Good -> Can focus TalkBack on list and correctly change item
* Needs adjustment -> Can focus TalkBack on the radio button (label is not identified)

Requested change:
* Similar to how we made the switch component before.
* Should be able to focus on the text/list element and change selection, radio button alone should not be focusable alone aka the list pair should be only focusable as one element.

Reply:
Done

Reply:
Tested in TalkBack, works great, thanks!